### PR TITLE
Update user README.md

### PR DESCRIPTION
**Pull request by vadekh**
## Related Issue

JUST PART:
https://github.com/digitallinguistics/spec/issues/301

## Description

Updated example language tags to IETF format.

## Changelog

UPDATE: update example language tags

## Pull Request Checklist

- [ ] **[Open an issue](https://github.com/digitallinguistics/spec/issues/new)** for the change, if one is not open already. **NOTE:** It is a good idea to wait until your suggested change is approved by a maintainer before writing any code.

- [ ] **Fork the repository** and clone it to your computer.

- [ ] **Create an issue branch** for the changes you are making. You should only make changes on this branch for the issue you are currently working on.

- [ ] **Install [Node.js](https://nodejs.org/en/)**. Use the latest LTS release. If you need to manage multiple versions of Node.js on your computer, install [nvm](https://github.com/creationix/nvm) or [nvm windows](https://github.com/coreybutler/nvm-windows).

- [ ] **Install project dependencies** by running `npm install` from the command line in the project root.

- [ ] **Update the project documentation** (`README.md`, `CONTRIBUTING.md`, etc.) to reflect the changes you plan to make, if applicable.

- [ ] **Write or update (failing) tests** for the changes you are going to make (if needed). Tests only need to be written for issues that cannot be checked using the example data in the schemas.

- [ ] **Make changes to the schemas or code.** The schemas are located in `/schemas/yaml`. Do not update the schemas in `schemas/json`, or your changes will be overwritten.

- [ ] **Increment the version number** for any schemas you made changes to.

- [ ] **Update the `"description"` field** of any schemas / properties that you change.

- [ ] **Update examples** in the affected schemas to reflect the changes you made.

- [ ] **Get the latest changes from the `master` branch** and resolve any conflicts that arise. Run the tests again.

- [ ] **Build the project** (`npm run build`)

  - Generates JSON versions of the schemas
  - Regenerates the project documentation

- [ ] **Run the tests** (`npm test`) and fix issues that arise. If you find an issue that is unrelated to the one you are working on, open a new issue for it.

- [ ] **Check the project documentation** for your changes

- [ ] **Open a pull request** from your issue branch into the `master` branch. See GitHub's advice on [How to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/).

- [ ] **Wait for a reviewer to provide feedback**

- [ ] **Address any changes requested by the reviewer**. While working on your pull request, you can write `[ci skip]` in each of your commit messages to skip automated testing, until you are ready to test your code.

- [ ] **Wait for your pull request to be merged.**

- [ ] **Delete your issue branch.**

- [ ] **Celebrate!!!** :fireworks: Your work is complete! Thank you for your valuable contribution to DLx!

### CODE REVIEW

**Comment by dwhieb** [comment]
So just to clarify: this PR is just updating the example in the readme, correct? You're doing the rest of it in a different PR? (Totally fine if so - just wasn't sure.)

File: .github/README.md [hunk 1]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
 ```json
 {
   "transcription": {
-    "spa": "hola me llamo Daniel",
+    "sp": "hola me llamo Daniel",
     "ipa": "ola me jamo dænjəl"
```
Comment (dwhieb) [inline .github/README.md]: Now that I see this change, it occurs to me we should also change the IPA transcription to `spa-fonipa` (which is the official language tag for transcribing Spanish in IPA).

```suggestion
    "es-fonipa": "ola me jamo dænjəl"
```

File: .github/README.md [hunk 2]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
 ```json
 {
   "transcription": {
-    "spa": "hola me llamo Daniel",
+    "sp": "hola me llamo Daniel",
```
Comment (dwhieb) [inline .github/README.md]: The language tag for Spanish is `es` or `spa`. (`es` is from ISO 639-1, and `spa` from 639-3, the latest version; both are still completely valid.) Let's use `es` (the official advice from IETF is to choose the shortest language tag possible).

**Comment by vadekh** [comment]
Yes this PR is just the README.md. I had to update it on GitHub online. The rest was done offline and is in another PR.

File: .github/README.md [hunk 3]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
   },
   "translation": {
-    "eng": "Hello, my name is Daniel.",
+    "en": "Hello, my name is Daniel.",
   },
   "words": [
     {
       "transcription": {
-        "spa": "hola",
+        "sp": "hola",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es": "hola",
```

File: .github/README.md (hunk 2 above)
Comment (vadekh) [inline .github/README.md]:
```suggestion
    "es": "hola me llamo Daniel",
```

File: .github/README.md [hunk 4]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
         "ipa": "ola"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es-fonipa": "ola"
```

File: .github/README.md [hunk 5]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
       },
       "translation": {
-        "eng": "hello"
+        "en": "hello"
       },
       "morphemes": [
         {
           "transcription": {
-            "spa": "hola",
+            "sp": "hola",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es": "hola",
```

File: .github/README.md [hunk 6]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
             "ipa": "ola"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es-fonipa": "ola"
```

File: .github/README.md [hunk 7]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
-            "eng": "hello"
+            "en": "hello"
           }
         }
       ]
     },
     {
       "transcription": {
-        "spa": "me",
+        "sp": "me",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es": "me",
```

File: .github/README.md [hunk 8]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
         "ipa": "me"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es-fonipa": "me"
```

File: .github/README.md [hunk 9]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
       },
       "translation": {
-        "eng": "me"
+        "en": "me"
       },
       "morphemes": [
         {
           "transcription": {
-            "spa": "me",
+            "sp": "me",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es": "me",
```

File: .github/README.md [hunk 10]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
             "ipa": "me"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es-fonipa": "me"
```

File: .github/README.md [hunk 11]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
-            "eng": "1sg.DO"
+            "en": "1sg.DO"
           }
         }
       ]
     },
     {
       "transcription": {
-        "spa": "llamo",
+        "sp": "llamo",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es": "llamo",
```

File: .github/README.md [hunk 12]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
         "ipa": "jamo"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es-fonipa": "jamo"
```

File: .github/README.md [hunk 13]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
       },
       "translation": {
-        "eng": "I call"
+        "en": "I call"
       },
       "morphemes": [
         {
           "transcription": {
-            "spa": "llam",
+            "sp": "llam",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es": "llam",
```

File: .github/README.md [hunk 14]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
             "ipa": "jam"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es-fonipa": "jam"
```

File: .github/README.md [hunk 15]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
           },
           "gloss": {
-            "eng": "call"
+            "en": "call"
           }
         },
         {
           "transcription": {
-            "spa": "o",
+            "sp": "o",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es": "o",
```

File: .github/README.md [hunk 16]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
             "ipa": "o"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es-fonipa": "o"
```

File: .github/README.md [hunk 17]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
-            "eng": "1sg.PRES.IND.SUBJ"
+            "en": "1sg.PRES.IND.SUBJ"
           }
         }
       ]
     },
     {
       "transcription": {
-        "spa": "Daniel",
+        "sp": "Daniel",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es": "Daniel",
```

File: .github/README.md [hunk 18]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
         "ipa": "dænjəl"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
        "es-fonipa": "dænjəl"
```

File: .github/README.md [hunk 19]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
       },
       "translation": {
-        "eng": "Daniel"
+        "en": "Daniel"
       },
       "morphemes": [
         {
           "transcription": {
-            "spa": "Daniel",
+            "sp": "Daniel",
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es": "Daniel",
```

File: .github/README.md [hunk 20]
```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
...
             "ipa": "dænjəl"
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
            "es-fonipa": "dænjəl"
```

File: .github/README.md [hunk 21]
```diff
@@ -161,7 +161,7 @@ JSON format is easy to learn. It consists of just a few simple rules:
...
 
-* Objects contain a list of properties (also called attributes or fields) and their values, both placed in double quotes `" "` and separated by a colon `:`. Pairs of properties and values are separated by commas `,`. In the example above, the Utterance has a property called `"transcription"`, and the value of that property is an Object containing transcriptions of the Utterance in different orthographies. For example, the `"spa"` property shows a transcription of the Utterance in standard Spanish orthography: `"Hola, me llamo Daniel."`.
+* Objects contain a list of properties (also called attributes or fields) and their values, both placed in double quotes `" "` and separated by a colon `:`. Pairs of properties and values are separated by commas `,`. In the example above, the Utterance has a property called `"transcription"`, and the value of that property is an Object containing transcriptions of the Utterance in different orthographies. For example, the `"sp"` property shows a transcription of the Utterance in standard Spanish orthography: `"Hola, me llamo Daniel."`.
```
Comment (vadekh) [inline .github/README.md]:
```suggestion
* Objects contain a list of properties (also called attributes or fields) and their values, both placed in double quotes `" "` and separated by a colon `:`. Pairs of properties and values are separated by commas `,`. In the example above, the Utterance has a property called `"transcription"`, and the value of that property is an Object containing transcriptions of the Utterance in different orthographies. For example, the `"es"` property shows a transcription of the Utterance in standard Spanish orthography: `"Hola, me llamo Daniel."`.
```

**Comment by dwhieb** [comment]
Not gonna worry about running build scripts, since you'll be doing that in #311.

**Pull request merged**